packages/parser/src/index.ts

```diff
@@ -1,7 +1,6 @@
 import type { APIDocument, ParserOptions } from './types.js';
 
-import $RefParser from '@apidevtools/json-schema-ref-parser';
-import _dereference from '@apidevtools/json-schema-ref-parser/dist/lib/dereference';
+import $RefParser, { dereferenceInternal } from '@apidevtools/json-schema-ref-parser';
```

Comment:
Noticed that they were exporting the dereference method we were using from the main export so I refactored this to not have us pull it out of the dist directory.